### ISSUE TITLE
Train on Google Cloud

### ISSUE THREAD

**Issue by jewl123** [zalandoresearch/flair#863]
Hey All,

I have been having a hell of a time trying to install flair and get flair to run on Google Cloud. 

I believe the source of the error is that flair necessitates python 3.6+ while Google only has a runtime of python 3.5.

However, I was able to run a full and successful flair training session previously and I have absolutely no clue how i was able to do so. 

For reference, the only successful run was executed on the Google A.I. Platform's Jupyter Notebook.

Thank you for the wonderful repository and for your generous assistance !

**Comment by stefan-it** [comment]
Hi @jewl123,

I checked the Python version in a Colab notebook and they're using Python 3.6.8.

On a `--image-project=ml-images --image-family=tf-1-13` instance, I could see that Python 3.5.3 is used on Debian 9.

I found a nice tutorial that describes how to install Python 3.6 from source for Debian 9:

https://www.rosehosting.com/blog/how-to-install-python-3-6-4-on-debian-9/

I think this should also work with your Google Cloud VM :)

**Comment by jewl123** [comment]
Thank you for the help!

I actually figured it out last night with a weird hack.

I created a google cloud instance with pytorch preinstalled and **without** GPUs.

I then pip installed flair and stopped the kernel after installation to add the necessary GPUs.

Once the kernel was restarted with the newly instated GPUs I then reinstalled fliar (although reinstall output says already satisfied, it seemed to be necessary).

Then flair worked wonderfully and as intended.

I hope this helps someone else :)
